documentation/modules/rn-2.4.adoc

```diff
@@ -0,0 +1,143 @@
+
+.OCP console plugin
+
+This version of {project-full} has replaced the stand-alone UI with a UI that is integrated in the OCP Console. The new UI operates as an OCP Console plugin that adds the sub-menu `Migration` to the navigation bar. It is implemented in version 2.4, disabling the old UI. You can enable the old UI by setting `feature_ui: true` in ForkliftController. link:https://issues.redhat.com/browse/MTV-427[(MTV-427)]
+
+.Skip certification option
+
+'Skip certificate validation' option was added to VMware and RHV providers. If checked, the provider's certificate will not be validated and the UI will not ask for specifying a CA certificate.
+
+.Only third-party certificate required
+
+When defining a RHV provider that set with third-party certificate, only the third-party certificate needs to be specified and MTV would retrieve the Manager CA certificate automatically.
+
+.Conversion of VMs with RHEL9 guest operating system
+
+For cold migrations from vSphere to the local OCP cluster, we use virt-v2v on RHEL 9 that supports conversion of virtual machines with RHEL 9 as the guest operating system. link:https://issues.redhat.com/browse/MTV-332[(MTV-332)]
+
+[id="known-issues-24_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.Deleting migration plan does not remove temporary resources
+
+Deleting a migration plan does not remove temporary resources such as importer pods, conversion pods, config maps, secrets, failed VMs and data volumes. +
+You must archive a migration plan before deleting it in order to clean up the temporary resources. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[(BZ#2018974)]
+
+.Unclear error status message for VM with no operating system
+
+The error status message for a VM with no operating system on the *Migration plan details* page of the web console does not describe the reason for the failure. link:https://bugzilla.redhat.com/show_bug.cgi?id=2008846[(BZ#22008846)]
+
+.Log archive file includes logs of a deleted migration plan or VM
+
+If you delete a migration plan and then run a new migration plan with the same name or if you delete a migrated VM and then remigrate the source VM, the log archive file created by the {project-short} web console might include the logs of the deleted migration plan or VM. link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764[(BZ#2023764)]
+
+.QEMU guest agent is not installed on migrated VMs
+
+The QEMU guest agent is not installed on migrated VMs. Workaround: Install the QEMU guest agent with a post-migration hook. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018062[(BZ#2018062)]
+
+.Migration of virtual machines with encrypted partitions fails during conversion
+
+Occurs for vSphere: Migrations from RHV and OpenStack don't fail, but the encryption key may be missing on the target OCP cluster.
+
+.RHV snapshots are not deleted after a successful migration
```

Comment:
@ahadas , do we need to also state that VMWare snapshots are not deleted after a successful migration?

Reply:
yeah, but for openstack is should be a bit different:
```
.Snapshots that are created during the migration in OpenStack are not deleted
The Migration Controller service does not delete snapshots that are created during the migration for source virtual machines in OpenStack automatically. Workaround: the snapshots can be removed manually on OpenStack. 
```